### ISSUE TITLE
I don´t see anything different with the german cab

### ISSUE THREAD

**Issue by MattIQ157** [aeughsendhelp/dumb282tweaks#17]
If I spawn a standart S282 and then enable german cab and spawn another loco right next to the first, I don´t see any differnces. Am I just not spotting them or is it a bug? I´m seeing other changes (like streamlined boiler)

**Comment by evanielt** [comment]
Yeah, its bugged rn. Im working on completely overhauling it, might be a little while before its all fixed
